README.md

```diff
@@ -4,7 +4,20 @@ This project was generated with [Angular CLI](https://github.com/angular/angular
 
 ## Development server
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+Run `npm start` for a dev server. This will run both angular server and the backend server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+
+## Production server
+
+You need to make sure you built the app before running it in production mode.
+run `npm run build` to build the app
+
+Run `npm run start:prod` to start both server and client code in production mode
+
+## Create a global link to ng-wiz
```

Comment:
I would like it if this was the only way we show to run the production as the user should use it inside his project or Dev folder.
I would want the user to only `npm install ngwiz` and then call it.

also if the link was to `ngWiz` (case insensitive) indtead of `ng-wiz`.

Reply:
you can't call it `ng-wiz` npm does not support dashes

Reply:
the Dev run is for developers who would like to help and contribute to the repo

Reply:
> the Dev run is for developers who would like to help and contribute to the repo

I meant a folder containing projects -> Development folder

Reply:
> you can't call it `ng-wiz` npm does not support dashes

@ortichon actually, it's the other way around. npm **does support dashes**. it doesn't support camelCase as a package name because the name field doesn't support uppercase letters.
Check out the [documentation](https://docs.npmjs.com/files/package.json#name).

Although, the executable can be named whatever you want.

But I would suggest that the package name and executable should be named the same (without the scope), or something completely different.

for example: `@angular/devkit-repo` [have an executable](https://github.com/angular/angular-cli/blob/master/package.json) named `ng` (and a bunch more)
but `ungit`'s [main executable](https://github.com/FredrikNoren/ungit/blob/master/package.json) is `ungit`


> I would like it if this was the only way we show to run the production as the user should use it inside his project or Dev folder.
> I would want the user to only `npm install ngwiz` and then call it.
> 
> also if the link was to `ngWiz` (case insensitive) instead of `ng-wiz`.

@ZimGil if I understood you correctly, you want this to behave similarly to `ungit`.

The behavior should be:
- you can run it anywhere.
   - if you run it inside an angular project, show that project
   - if you run it inside a development folder, show all projects (I guess that's already supported)
- the package should be able to run either by
   - installing the package globally with `npm install -g ng-wiz`
   - or as a dev dependency of another package with `npm install --save-dev ng-wiz`

I basically messed up the documentation a little (which I'll fix), but with this change, all those requirements should be met.

- ✔️ you can run it anywhere
   > if you install this globally with `npm install -g ng-wiz`, this will work out of the box since `cwd` will point to were you ran the program from
- ✔️ the package should be able to run either by
   - ✔️ installing the package globally with `npm install -g ng-wiz`
      > with the `bin` field in the `package.json` + the executable `ng-wiz.js`, this is exactly the behavior you'll get when installing this globally. Basically, it will run `ng-wiz.js` when executed
   - ✔️ or as a dev dependency of another package with `npm install --save-dev ng-wiz`
      > when installing a package as a dependency, the `bin` field is checked as well. If the `bin` field is present, the package is installed **and an executable link is created in the `node_modules/.bin` folder** which will point, again, to `ng-wiz.js`. Whenever a user tries to run any package that is installed locally and has an executable, basically they either run that package from the `package.json` script field or by running `npx ng-wiz`. both **check automatically for the executables in the `.bin` folder**

I'll change the documentation I wrote in `readme.md` to be clearer and also add some examples to the PR description so you can test it and make sure it works as expected

Reply:
and for clarification: `npm link` is just a development tool to test a package as if it was installed globally. basically, it links the local package executable file (here it's `bin/ng-wiz.js`) to the global npm executables. Users of this module shouldn't really do that, only developers

Reply:
@ZimGil updated the PR description and made documentation on how to use vs. how to develop to reflect better what I did.

Also added a code of conduct and structured the contributing file after [angular's contributing file](https://github.com/angular/angular/blob/master/CONTRIBUTING.md).

Code of conduct was generated by the automatic process suggested by GitHub